driverkit/Makefile

```diff
@@ -5,12 +5,16 @@ DRIVERKIT ?= $(shell which driverkit)
 CONFIGS := $(wildcard config/*/*.yaml)
 VERSIONS := $(patsubst config/%,%,$(sort $(dir $(wildcard config/*/))))
 VERSIONS := $(VERSIONS:/=)
-TARGET_DISTRO ?=
+TARGET_DISTRO ?= *
```

Comment:
Not sure we need `TARGET_DISTRO` to equal to `*` by default.

In case you need to build all you can use `make all`

Reply:
I was thinking about it in case we want to filter only by kernel version, but maybe it's not useful?